src/main/java/com/team4/testingsystem/repositories/CoachGradeRepository.java

```diff
@@ -0,0 +1,21 @@
+package com.team4.testingsystem.repositories;
+
+import com.team4.testingsystem.entities.CoachGrade;
+import com.team4.testingsystem.entities.Question;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+import javax.transaction.Transactional;
+
+@Repository
+public interface CoachGradeRepository extends CrudRepository<CoachGrade, Long> {
+    Optional<CoachGrade> findByQuestion(Question question);
```

Comment:
A question can be equal in different records.
I think we need to find grades by test if we want to get both of the grades (expected), or by test and question if (for some reason) we are cares about a single mark